app/src/main/res/drawable/ic_gas_flame.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
```

Comment:
Added the icon details to the xml field

Reply:
Added the XML file as it I needed to add the icon so had to put in details there

Reply:
https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-2-user-experience/lesson-4-user-interaction/4-1-c-buttons-and-clickable-images/4-1-c-buttons-and-clickable-images.html